templates/skeleton/app/routes/account.addresses.tsx

```diff
@@ -20,7 +20,7 @@ import {
   UPDATE_ADDRESS_MUTATION,
   DELETE_ADDRESS_MUTATION,
   CREATE_ADDRESS_MUTATION,
-} from 'app/graphql/customer-account/CustomerAddressMutations';
+} from '~/graphql/customer-account/CustomerAddressMutations';
```

Comment:
@frandiox interesting. this is necessary for the generator or just done out of consistency?